mora/service/orgunit.py

```diff
@@ -424,119 +506,157 @@ def edit_org_unit(unitid):
 
     :statuscode 200: The edit succeeded.
 
-    **Example Request**:
-
     :param unitid: The UUID of the organisational unit.
```

Comment:
Update the docstring of this function to make it clear that we are now editing the details for an org unit, and no longer just the org unit itself.

Reply:
Indeed, and I'll fix the other docstring as well while at it!